main.cpp

```diff
@@ -407,6 +382,29 @@ FloatType& FloatType::powInternal(float f)
     return *this;
 }
 
+FloatType& FloatType::apply(std::function<FloatType&(float&)> func)
+{
+    if(func)
+    {
+        return func(*value);
+    }
+    return *this;
+}
+
+FloatType& FloatType::apply(void(*func)(float&))
+{
+    if(func)
+        {
+            func(*value);
+        }
+        return *this;
+}
```

Comment:
```
## 4) Wait for me to review your work.
...
While you're waiting, view the Pull Request Files tab.
This will show you all of the changes in your pull request.  
You'll see exactly what I see, and you might even spot some things that you missed when looking at your project source files, like formatting issues.
```

Fix this formatting

- [ ] revise